test/core/test_dataarray.py

```diff
@@ -6,34 +6,33 @@
 from uxarray.core.dataset import UxDataset, UxDataArray
 import pytest
 
-current_path = Path(os.path.dirname(os.path.realpath(__file__)))
+# Import centralized paths
+import sys
+sys.path.append(str(Path(__file__).parent.parent))
+from paths import *
 
-gridfile_ne30 = current_path / "meshfiles" / "ugrid" / "outCSne30" / "outCSne30.ug"
-dsfile_var2_ne30 = current_path / "meshfiles" / "ugrid" / "outCSne30" / "outCSne30_vortex.nc"
 
-gridfile_geoflow = current_path / "meshfiles" / "ugrid" / "geoflow-small" / "grid.nc"
-dsfile_v1_geoflow = current_path / "meshfiles" / "ugrid" / "geoflow-small" / "v1.nc"
 
 def test_to_dataset():
     """Tests the conversion of UxDataArrays to a UXDataset."""
-    uxds = ux.open_dataset(gridfile_ne30, dsfile_var2_ne30)
-    uxds_converted = uxds['psi'].to_dataset()
+    uxds = ux.open_dataset(OUTCSNE30_GRID, OUTCSNE30_VAR2)
```

Comment:
Use fixture

Reply:
I only changed on file for now test/core/test_accessors.py - confirming things before changes to all files